tests/wpt/metadata/FileAPI/blob/Blob-slice.html.ini

```diff
@@ -1,62 +1,62 @@
 [Blob-slice.html]
   type: testharness
   [Slicing test: slice (5,0).]
-    expected: FAIL
+    expected: PASS
```

Comment:
When we'd like to make the test as `PASS`, we can just remove the `expected: FAIL` to make it as `PASS`

Ex.
```diff
[Blob-slice.html]
   type: testharness
-   [Slicing test: slice (5,0).]
-     expected: FAIL
-     bug: https://github.com/servo/servo/issues/10911
```

Reply:
Ok let me fix it